### ISSUE TITLE
Support for INNER JOIN with values from other table in SELECT?

### ISSUE THREAD

**Issue by hardartcore** [satyan/sugar#237]
I've read the documentation, I've check the source code and I didn't find anything if I can use something like this:
`SELECT a.article_id, l.list_title FROM articles AS a INNER JOIN user_list AS l ON (l.article_id = a.article_id) WHERE l.list_id = ?`

**Comment by whoshuu** [comment]
You're right, there's no inner join support right now. As it stands, Sugar's interface to the underlying SQLite database is based around object models. We're going to support a basic cursor return, and if you understand how the cursor works, you can use an inner join there.
